cacti/tests/test_cacti.py

```diff
@@ -0,0 +1,170 @@
+    'rra[3].cdp_prep[0].unknown_datapoints': 177L,
+    'rra[7].pdp_per_row': 288L,
+    'rra[1].xff': 0.5,
+    'rra[3].cur_row': 100L,
+    'rra[3].cdp_prep[0].value': 83282.52,
+    'rra[2].cf': 'AVERAGE',
+    'rra[4].cur_row': 454L,
+    'rra[2].cdp_prep[0].value': 83282.52,
+    'ds[mem_buffers].minimal_heartbeat': 600L,
+    'rra[0].cdp_prep[0].unknown_datapoints': 0L,
+    'rra[5].cdp_prep[0].unknown_datapoints': 0L,
+    'ds[mem_buffers].type': 'GAUGE',
+    'rra[1].pdp_per_row': 6L,
+    'rra[1].rows': 700L,
+    'rra[3].rows': 797L,
+    'rrd_version': '0003',
+    'rra[7].xff': 0.5,
+    'rra[6].rows': 775L,
+    'filename': '/var/lib/cacti/rra/localhost_mem_buffers_3.rrd',
+    'rra[1].cf': 'AVERAGE',
+    'last_update': 1484061001L,
+    'rra[5].rows': 700L,
+    'rra[4].xff': 0.5,
+    'rra[0].cf': 'AVERAGE',
+    'ds[mem_buffers].index': 0L,
+    'rra[4].cf': 'MAX',
+    'rra[3].pdp_per_row': 288L,
+    'ds[mem_buffers].last_ds': '20376',
+    'rra[7].cdp_prep[0].value': 21262.533333333333,
+    'rra[2].xff': 0.5,
+    'ds[mem_buffers].unknown_sec': 0L,
+    'rra[0].pdp_per_row': 1L,
+    'rra[7].rows': 797L,
+    'rra[0].cur_row': 42L,
+    'rra[6].xff': 0.5,
+    'rra[4].pdp_per_row': 1L,
+    'header_size': 2040L,
+    'rra[5].pdp_per_row': 6L,
+    'rra[7].cf': 'MAX',
+    'step': 300L,
+    'rra[3].xff': 0.5,
+    'ds[mem_buffers].max': None,
+    'rra[6].cdp_prep[0].unknown_datapoints': 9L,
+    'rra[4].cdp_prep[0].value': None,
+    'ds[mem_buffers].min': 0.0,
+    'rra[0].cdp_prep[0].value': None,
+    'rra[1].cdp_prep[0].value': 41644.66666666667,
+    'rra[1].cur_row': 234L,
+    'rra[0].rows': 600L,
+    'rra[6].cdp_prep[0].value': 21262.533333333333,
+    'rra[5].cur_row': 194L,
+    'rra[4].rows': 600L,
+    'rra[3].cf': 'AVERAGE',
+    'rra[0].xff': 0.5,
+    'rra[5].cf': 'MAX',
+    'rra[2].cdp_prep[0].unknown_datapoints': 9L,
+    'rra[5].xff': 0.5,
+    'rra[2].cur_row': 742L,
+    'rra[6].cf': 'MAX',
+    'rra[7].cdp_prep[0].unknown_datapoints': 177L,
+    'rra[1].cdp_prep[0].unknown_datapoints': 0L,
+    'rra[2].rows': 775L,
+    'rra[2].pdp_per_row': 24L,
+}
+
+ts = int(time.time())
+MOCK_FETCH = ((ts - 300, ts, 300), ('mem_buffers',), [(2048,), (None,)])
+
+MOCK_RRD_META = [
+    ('localhost', None, '/var/lib/cacti/rra/localhost_mem_buffers_3.rrd'),
+    ('localhost', None, '/var/lib/cacti/rra/localhost_mem_swap_4.rrd'),
+    ('localhost', None, '/var/lib/cacti/rra/localhost_load_1min_5.rrd'),
+    ('localhost', None, '/var/lib/cacti/rra/localhost_users_6.rrd'),
+    ('localhost', None, '/var/lib/cacti/rra/localhost_proc_7.rrd'),
+]
+
+
+CHECK_NAME = 'cacti'
+
+CACTI_CONFIG = {
+    'mysql_host': 'nohost',
+    'mysql_user': 'mocked',
+    'rrd_path': '/rrdtool/is/mocked',
+    'tags': ['optional:tag1'],
+}
+
+
+@pytest.fixture
+def aggregator():
+    from datadog_checks.stubs import aggregator
+    aggregator.reset()
+    return aggregator
+
+
+@pytest.fixture
+def check():
```

Comment:
This is more of a style thing, but can we not use the name ``check`` as the fixture name? It could be, ``cacti_check`` or just ``cacti``. I'm just trying to avoid ``check.check(...)``.

Reply:
i don't know, it doesn't shock me, plus it's already like this in many checks

Reply:
Fair enough 👍